File: Javascript/Math/Count-Odd-Numbers-In-An-Interval-Range.js
/**
 *
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
const countOdds = (low, high) => {
  let count = 0;

  for (let i = low; i <= high; i++) {
    if (i % 2 !== 0) count++;
  }

  return count;
};

File: Rust/Count-Odd-Numbers-In-An-Interval-Range.rs
pub struct Solution;

impl Solution {
    pub fn count_odds(low: i32, high: i32) -> i32 {
        let mut result = (high - low) / 2;

        if low % 2 != 0 || high % 2 != 0 {
            result += 1;
        }

        result
    }
}

fn main() {}
